pep-0716.rst

```diff
@@ -0,0 +1,319 @@
+was designed to be a normalization applied when two computers spoke to each
+other, not as something that would "leak" out into the human-facing areas.
+
+
+Specification
+=============
+
+The project name that is specified by an author ends up flowing through several
+parts of the ecosystem, and each part needs to be considered on its own to determine what
+kind of name (normalized or not) makes sense in that part.
+
+In general, we follow the guiding principals, use the unnormalized name as
+provided by the author wherever possible, and normalize strictly where not.
+
+In some cases, we are simply repeating the status quo, this is done to provide
+clarification and to be explicit which uses were considered as part of this
+PEP.
+
+
+Core Metadata
+-------------
+
+The ``Name`` field **MUST NOT** be normalized when emitting into ``METADATA``
+or ``PKG-INFO``.
+
+The ``Name`` field **MUST NOT** be normalized when uploading to a repository.
+
+The ``Name`` field **SHOULD NOT** be normalized when being presented for display
+to a user.
+
+The ``Name`` field **MUST** be normalized during comparison.
+
+Tools that read the ``Name`` field from a core metadata file **MUST** be prepared
+to accept unnormalized names.
+
+
+pyproject.toml
+--------------
+
+The ``project.name`` key **MUST** be preserved exactly as the author chose to
+represent it, and **MUST** be emitted in this way into ``METADATA`` or
+``PKG-INFO``.
+
+The ``project.name`` field **MUST** be normalized during comparison.
+
+
+.dist-info directories
+----------------------
+
+The directory name follows the pattern of ``{name}-{version}.dist-info``.
+
+The ``name`` field **MUST** be normalized, with any resulting ``-`` escaped to ``_``.
+
+Tools that read an arbitrary ``.dist-info`` directory **MUST** be prepared to
+accept unnormalized values, however tools that work only on *new* ``.dist-info``
+directories **SHOULD** validate that all values are normalized.
+
+
+Source and Binary Distributions
+-------------------------------
+
+Both the sdist and bdist specifications incorporate the project name in their
+filenames (``{name}-{version}.tar.gz`` and
+``{distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl``
+respectively).
+
+The ``name`` field **MUST** be non-normalized, with the exception that any ``-``
+**MUST** be escaped to be ``_``.
+
+Tools that accept an arbitrary distribution **MUST** be prepared to accept both
+non-normalized and normalized filenames. However, tools that only work on *new*
+distributions **SHOULD** validate that the distribution filenames are not
+normalizing ``name``.
+
+
+Simple Repository API
+---------------------
+
+The project name, when returned in the "index" URL (e.g. ``/simple/``)
+**MUST** be non-normalized.
+
+The project name when used in the URL (e.g. ``/simple/$project/``) **MUST** be
+normalized.
+
+The project name, when used on the Project detail page
```

Comment:
In the HTML version of the index, the project name is never used (except in any additional content not covered by the spec).  So this requirement can be omitted - unless you want to cover the JSON form as defined in PEP 691, in which case

```suggestion
The project name, when referenced in the `"name"` field of the JSON format project detail page
```

Reply:
(Fixed mistaken single backticks in the above comment)